tests/unit/test_mup_optimizers.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
What folder should this go into inside of `tests/unit`?

Reply:
Probably `unit/runtime/`, but maybe in one of the subdirs. Look for where we have other tests related to optimizers!

Reply:
Moved to runtime.  The other optimizers are ops, so they are in folders there.  So just moving this to runtime for now.